std/csv.d

```diff
@@ -1425,7 +1425,7 @@ if (isSomeChar!Separator && isInputRange!Range
 
     while (!input.empty)
     {
-        assert(!(quoted && escQuote));
+        assert(!(quoted && escQuote), "Invalid quotation state");
```

Comment:
Consider adding the function to the message in some fashion. This is often helpful when debugging. eg.
```
"Invalid quotation state in csvNextToken"
```

Reply:
Just realized this is part of a much larger effort to add messages to all asserts. (Fantastic!) So, ignore my comment about this specific case and instead be consistent across all cases.

FWIW, I've benefitted when the assert messages include a reference to the code construct that is asserting. This has even been helpful to tracking down bugs in Phobos. But most important is that there is an assert message.